modules/claims_api/app/controllers/claims_api/v0/forms/base.rb

```diff
@@ -72,13 +72,6 @@ def decode_document(document)
                                                  tempfile: temp_file)
         end
 
-        def bgs_service
```

Comment:
moving this up a level to the application controller so it's also accessible from the claims controller